src/overlay/TCPPeer.h

```diff
@@ -17,6 +17,9 @@ namespace stellar
 {
 
 static auto const MAX_UNAUTH_MESSAGE_SIZE = 0x1000;
+// action quee name must be uniform across all postOnMainThread calls to ensure
```

Comment:
```suggestion
// action queue name must be uniform across all postOnMainThread calls to ensure
```

Reply:
This code went away after the latest refactor, so I'm resolving this.